.gitignore

```diff
@@ -2,3 +2,4 @@ bin/
 gopath/
 *.sw[ponm]
 .DS_Store
+stage1.aci
```

Comment:
I always run rkt as bin/rkt during development.  I'll move it into bin if you think it's more appropriate.  The stage1.aci isn't an executable though, which is true for everything else we put in bin.  Since I build via ./build, I tend to run bin/rkt which expects stage1.aci where I have it.

Maybe the default should be an absolute path derived at build time so I can continue running `bin/rkt` and others `./rkt` so everyone's happy.  Is there an equivalent to `gcc -DSTAGE1_PATH=/foo/bar.aci` for `go build`?